src/apps/umami/arbitrum/umami.compound.token-fetcher.ts

```diff
@@ -16,13 +18,42 @@ const appId = UMAMI_DEFINITION.id;
 const groupId = UMAMI_DEFINITION.groups.compound.id;
 const network = Network.ARBITRUM_MAINNET;
 
+type UmamiMarinateApiObject = {
+  apy: string;
+};
+
+type UmamiCompounderApiObject = {
+  tvl: number;
+};
+
+export type UmamiApiDatas = {
+  marinate: UmamiMarinateApiObject;
+  mUmamiCompounder: UmamiCompounderApiObject;
+};
+
 @Register.TokenPositionFetcher({ appId, groupId, network })
 export class ArbitrumUmamiCompoundTokenFetcher implements PositionFetcher<AppTokenPosition> {
   constructor(
     @Inject(APP_TOOLKIT) private readonly appToolkit: IAppToolkit,
     @Inject(UmamiContractFactory) private readonly umamiContractFactory: UmamiContractFactory,
   ) {}
 
+  @CacheOnInterval({
+    key: `apps-v3:${network}:${appId}:${groupId}:informations`,
```

Comment:
Replace `apps-v3` with `studio` ("apps-v3" is a legacy term used internally at Zapper)